frontend/src/scenes/web-analytics/webAnalyticsLogic.tsx

```diff
@@ -1714,6 +1401,11 @@ export const webAnalyticsLogic = kea<webAnalyticsLogicType>([
                 filter_test_accounts,
             }
         ) => {
+            if (values.hasLoadedStateFromUrl) {
+                // only load from URL once, to prevent infinite loops
+                return
+            }
```

Comment:
It feels wrong having to do this 🤔

Looking through the codebase it doesn't seem to be the case that we have to do this elsewhere. `urlToAction` to seems to protect itself somehow, but now that I think about it I'm not sure how

Reply:
I don't need it - we don't get infinite loops without it, maybe I'm just being paranoid

Reply:
I've removed it